python/cucim/src/cucim/skimage/segmentation/_chan_vese.py

```diff
@@ -65,7 +67,7 @@ def _fused_variance_kernel2(
     difference_term += term2
 
     new_phi = phi + (dt * delta_phi) * (mu * K + difference_term)
-    out = new_phi / (1 + mu * dt * delta_phi * Csum)
+    out = new_phi / (one + mu * dt * delta_phi * Csum)
```

Comment:
If we wanted to generalize the type handling here, we could do something like this. That way we promote `1` to the same `dtype` of the result it is being added to

```suggestion
    out_denom = mu * dt * delta_phi * Csum
    out_denom += out_denom.dtype.type(1)
    out = new_phi / out_denom
```

Reply:
Thanks, pushed a new commit following this suggestion